Makefile

```diff
@@ -16,7 +16,7 @@ test4:
 devdeps:
 	pip install --upgrade pip setuptools
 	pip install wheel twine
-	pip install 'black==22.3' 'snakefmt==0.6' 'pytest>=6.0' pytest-cov pytest-xdist myst-parser sphinx sphinx-argparse
+	pip install 'black<25.0.0,>=24.1.1' 'snakefmt>=0.10.0' 'pytest>=6.0' pytest-cov pytest-xdist myst-parser sphinx sphinx-argparse
```

Comment:
`snakefmt==0.6` required `importlib-metadata <2.0.0,>=1.7.0`  but `multiqc` requires version >=7.1.0. This was the source of the CI build failures. Upgrading the `snakefmt` and `black` versions resolves this issue.